module/VuFind/tests/unit-tests/src/VuFindTest/RecordTab/MapTest.php

```diff
@@ -0,0 +1,191 @@
+<?php
+/**
+ * Map Test Class
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2022.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Sudharma Kellampalli <skellamp@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+namespace VuFindTest\RecordTab;
+
+use VuFind\RecordTab\Map;
+
+/**
+ * Map Test Class
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Sudharma Kellampalli <skellamp@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+class MapTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * Test getting Description.
+     *
+     * @return void
+     */
+    public function testGetDescription(): void
+    {
+        $obj = new Map();
+        $expected = 'Map View';
+        $this->assertSame($expected, $obj->getDescription());
+    }
+
+    /**
+     * Test if the tab loaded via AJAX.
+     *
+     * @return void
+     */
+    public function testSupportsAjax(): void
+    {
+        $obj = new Map();
+        $this->assertFalse($obj->supportsAjax());
+    }
+
+    /**
+     * get a Map object
+     *
+     * @return Map
+     */
+    public function getMap()
```

Comment:
Would it make sense to use getMap everywhere for consistency (instead of instantiating Map directly in the two tests above) or else to rename this method getConfiguredMap to better explain its purpose?